manageiq-api-common.gemspec

```diff
@@ -16,6 +16,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "acts_as_tenant"
   spec.add_runtime_dependency "manageiq-password", "~>0.1"
   spec.add_runtime_dependency "pg", "> 0"
+  spec.add_runtime_dependency "prometheus-client", "~> 0.8.0"
```

Comment:
Should we visually separate this from the rest of the runtime dependencies and note which optional feature it's here for?

Reply:
If that's something we're going to start doing then I'm fine with it, but isn't tenancy an optional feature also?

Reply:
It's not opt-in at the moment, but it probably should be.  And I think that dependency should be visually separate also.